src/screens/HomeScreen.js

```diff
@@ -0,0 +1,242 @@
+import React from "react";
+import "../App.css";
+import { useState, useEffect } from "react";
+import axios from "axios";
+import DUMMY_DATA from "../data/boards.json";
+import DUMMY_CARDS from "../data/card.json";
+import NewBoardForm from "../components/NewBoardForm";
+import BoardsList from "../components/BoardsList";
+import CardsForSelectedBoard from "../components/CardsForSelectedBoard";
+import CreateNewCard from "../components/CreateNewCard";
+import SortOption from "../components/SortOption";
+
+const kBaseUrl = process.env.REACT_APP_BACKEND_URL;
+
+const transformCardResponse = (card) => {
+  const {
+    id,
+    message,
+    likes_count: likesCount,
+    board_id: boardId,
+    status,
+  } = card;
+  return { id, message, likesCount, boardId, status };
+};
+
+const INIT_DATA = DUMMY_DATA.map((board) => {
+  const reformedCards = board.cards.map((card) => transformCardResponse(card));
+  return { ...board, cards: reformedCards };
+});
+const INIT_CARDS = DUMMY_CARDS.map((card) => transformCardResponse(card));
+
+const getAllBoards = () => {
+  return axios
+    .get(`${kBaseUrl}/boards`)
+    .then((response) => {
+      return response.data;
+    })
+    .catch((error) => {
+      console.log(error);
+      return INIT_DATA; //this will be deleted after connecting with Back-end
+    });
+};
+
+const getCardsForSelectedBoard = (id) => {
+  return axios
+    .get(`${kBaseUrl}/boards/${id}/cards`)
+    .then((response) => {
+      console.log(response.data.map(transformCardResponse));
```

Comment:
Reminder that we should remove debugging statements when done with functionality/before opening PRs so they do not get committed into production code.